src/3_branch.js

```diff
@@ -1179,11 +1179,11 @@ if (WEB_BUILD) { // jshint undef:false
  * Remove the listener from observations, if it is present. Not that this function must be passed a referrence to the _same_ function that was passed to `branch.addListener()`, not just an identical clone of the function.
  *
  */
-/*** +TOC_ITEM #removelistener &.removeListener()& ^WEB ***/
+/*** +TOC_ITEM #removelistenerlistener &.removeListener()& ^WEB ***/
 	Branch.prototype['removeListener'] = function(listener) {
 		if (listener) {
 			this._listeners = this._listeners.filter(function(subscription) {
-				if (subscription.listener !== listener) { return subscription; }
+				if (subscription["listener"] !== listener) { return subscription; }
```

Comment:
@dmitrig01 I can't remember how to do this... it's not externs is it? Can you remember the name?